examples/rtc_s1_sample_runconfig-v2.0.0.yaml

```diff
@@ -163,16 +165,14 @@ RunConfig:
                         # This should match the path used for OutputProductPath
                         output_dir: /home/rtc_user/output_dir
 
-                        # These field determines the intermediate file name used
-                        # by the SAS for its output products. These products will
-                        # be renamed by the PGE to match the OPERA file name conventions.
-                        product_id: OPERA_L2_RTC-S1_{burst_id}
-
                         # Validity start date for RTC-S1-STATIC products in the format YYYYMMDD,
                         # This field must be provided for RTC-S1 jobs when static layer
                         # generation is enabled, and should match the value assigned to
                         # DataValidityStartDate above.
-                        rtc_s1_static_validity_start_date: 20140101
+                        rtc_s1_static_validity_start_date: 20140403
+
+                        # Location from where the output product can be retrieved (URL or DOI)
```

Comment:
Just curious.  This value matches 'source_data_access' under 'input_file_group'.  If this will always be the case (which I imagine it will not be, but just throwing it out there), it might make sense to factor it out into another key.